CI: Runtime Test (privileged): Test_LBEgressAdvertisementWithLoadBalancerIP

## CI failure

```
┏━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┓
┃   FAIL  package: github.com/cilium/cilium/pkg/bgpv1/test   ┃
┗━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┛
--- FAIL: Test_LBEgressAdvertisementWithLoadBalancerIP (15.06s)
time="2024-11-29T08:58:05Z" level=info msg="deleting dummy links"
time="2024-11-29T08:58:05Z" level=info msg="adding dummy links"
time="2024-11-29T08:58:05Z" level=info msg="GoBGP test instance: starting"
time="2024-11-29T08:58:05Z" level=info msg="Add a peer configuration" Key=172.16.100.1 Topic=Peer asn=65011 component=tests.BGP subsys=basic
time="2024-11-29T08:58:05Z" level=debug msg="IdleHoldTimer expired" Duration=0 Key=172.16.100.1 Topic=Peer asn=65011 component=tests.BGP subsys=basic
time="2024-11-29T08:58:05Z" level=debug msg="state changed" Key=172.16.100.1 Topic=Peer asn=65011 component=tests.BGP new=BGP_FSM_ACTIVE old=BGP_FSM_IDLE reason=idle-hold-timer-expired subsys=basic
    logger.go:257: time=2024-11-29T08:58:05.608Z level=INFO source=/host/vendor/github.com/cilium/hive/hive.go:338 msg=Starting
time="2024-11-29T08:58:05.612334422Z" level=info msg="Using v1.Endpoints" subsys=k8s
    logger.go:257: time=2024-11-29T08:58:05.615Z level=INFO source=/host/vendor/github.com/cilium/hive/hive.go:342 msg=Started duration=4.573758ms
time="2024-11-29T08:58:05Z" level=info msg="GoBGP test instance: Peer Event: {0 IDLE}"
time="2024-11-29T08:58:05Z" level=info msg="GoBGP test instance: Peer Event: {65001 IDLE}"
time="2024-11-29T08:58:05Z" level=info msg="GoBGP test instance: Peer Event: {65001 ACTIVE}"
time="2024-11-29T08:58:05.812793497Z" level=info msg="Cilium BGP Control Plane Controller now running..." component=Controller.Run subsys=bgp-control-plane
time="2024-11-29T08:58:05.815762089Z" level=info msg="Registering BGP servers for policy with local ASN 65001" component=manager.registerBGPServer subsys=bgp-control-plane
time="2024-11-29T08:58:05.820338122Z" level=info msg="Successfully registered GoBGP servers for policy with local ASN 65001" component=manager.registerBGPServer subsys=bgp-control-plane
time="2024-11-29T08:58:11Z" level=debug msg="try to connect" Key=172.16.100.1 Topic=Peer asn=65011 component=tests.BGP subsys=basic
time="2024-11-29T08:58:11.611572973Z" level=info msg="Can't find configuration for a new passive connection" Key=172.16.100.2 Topic=Peer asn=65001 component=gobgp.BgpServerInstance subsys=bgp-control-plane
time="2024-11-29T08:58:11Z" level=debug msg="state changed" Key=172.16.100.1 Topic=Peer asn=65011 component=tests.BGP new=BGP_FSM_OPENSENT old=BGP_FSM_ACTIVE reason=new-connection subsys=basic
time="2024-11-29T08:58:11Z" level=debug msg="state changed" Key=172.16.100.1 Topic=Peer asn=65011 component=tests.BGP new=BGP_FSM_IDLE old=BGP_FSM_OPENSENT reason=read-failed subsys=basic
time="2024-11-29T08:58:11Z" level=info msg="GoBGP test instance: Peer Event: {65001 OPENSENT}"
time="2024-11-29T08:58:11Z" level=info msg="GoBGP test instance: Peer Event: {65001 IDLE}"
time="2024-11-29T08:58:16Z" level=debug msg="IdleHoldTimer expired" Duration=5 Key=172.16.100.1 Topic=Peer asn=65011 component=tests.BGP subsys=basic
time="2024-11-29T08:58:16Z" level=debug msg="state changed" Key=172.16.100.1 Topic=Peer asn=65011 component=tests.BGP new=BGP_FSM_ACTIVE old=BGP_FSM_IDLE reason=idle-hold-timer-expired subsys=basic
time="2024-11-29T08:58:16Z" level=info msg="GoBGP test instance: Peer Event: {65001 ACTIVE}"
    adverts_test.go:685: 
        	Error Trace:	/host/pkg/bgpv1/test/adverts_test.go:685
        	Error:      	Received unexpected error:
        	            	did not receive expected peering state ["ESTABLISHED"]: context deadline exceeded
        	Test:       	Test_LBEgressAdvertisementWithLoadBalancerIP
```

Job: https://github.com/cilium/cilium/actions/runs/12077530924/job/33690883378
(no sysdumps produced by the job)

--- a/pkg/bgpv1/test/adverts_test.go
+++ b/pkg/bgpv1/test/adverts_test.go
@@ -26,10 +26,7 @@ import (
 
 var (
 	// maxTestDuration is allowed time for test execution
-	maxTestDuration = 25 * time.Second
-
-	// maxGracefulRestartTestDuration is max allowed time for graceful restart test
-	maxGracefulRestartTestDuration = 1 * time.Minute
+	maxTestDuration = 5 * time.Minute
 )
 
 // Test_PodCIDRAdvert validates pod IPv4/v6 subnet is advertised, withdrawn and modified on node addresses change.

--- a/pkg/bgpv1/test/neighbor_test.go
+++ b/pkg/bgpv1/test/neighbor_test.go
@@ -264,7 +264,7 @@ func Test_NeighborGracefulRestart(t *testing.T) {
 	}
 
 	// This test run can take upto a minute
-	testCtx, testDone := context.WithTimeout(context.Background(), maxGracefulRestartTestDuration)
+	testCtx, testDone := context.WithTimeout(context.Background(), maxTestDuration)
 	defer testDone()
 
 	// test setup, we configure single gobgp instance here.

